utility/application/upload_service.py

```diff
@@ -16,7 +16,17 @@ def store_file(self, upload_type, file_data, request_params, username):
         """
             TODO: persist user history of the storage request
         """
-        if upload_type == UploadType.ORG_ASSETS.value:
+        if upload_type == UploadType.FEEDBACK.value:
+            bucket = UPLOAD_TYPE_DETAILS[upload_type]["bucket"]
+            dest_file_path = UPLOAD_TYPE_DETAILS[upload_type]["bucket_path"]\
+                .format(date_time_for_filename(), file_data["file_extension"])
+
+            self.boto_utils.s3_upload_file(file_data["file_path"], bucket, dest_file_path)
```

Comment:
Can this timeout ?

Reply:
if it exceeds the API gateway timeout 30 sec lambda (upload lambda) will stop and API gateway will throw error 504 timeout.

Reply:
Hmm but what if a file takes more than 30 secs to upload? Are we sure we wont run into that situation? Should we make this an sync call